core/templates/dev/head/pages/splash.html

```diff
@@ -155,7 +155,7 @@ <h2 class="oppia-splash-h2" translate="I18N_SPLASH_SUBTITLE"></h2>
       <div class="oppia-splash-section-two-content">
         <div class="oppia-splash-bullet">
           <div class="oppia-splash-bullet-block oppia-splash-block-left-image">
-            <img src="/images/splash/bullet1icon.svg">
+            <img src="{{get_static_resource_url('/assets/images/splash/bullet1icon.svg')}}">
```

Comment:
ditto (and elsewhere) -- use ng-src and angular interpolation. We're moving away from jinja.